src/parcel.js

```diff
@@ -0,0 +1,142 @@
+/* This import statement requires a peer or dev dependency on react that is fulfilled at runtime.
+ * To avoid duplicate bundling of react, we do not do this inside of single-spa-react.js.
+ * We also do not set up the prop types in this file to avoid requiring the user of the library
+ * to have prop-types installed and in their browser bundle, since not everyone uses prop types.
+ */
+import React from 'react'
+import {SingleSpaContext} from '../lib/single-spa-react.js'
+
+export default class Parcel extends React.Component {
+  static defaultProps = {
+    wrapWith: 'div'
```

Comment:
this is the default, correct?

Reply:
Yep, you can change it by passing in the `wrapWith` prop as `span` or something else